storage/src/lib.rs

```diff
@@ -703,29 +703,35 @@ pub mod tests_common {
 
             // create common RocksDB block cache to be shared among column families
             let db_cache = Cache::new_lru_cache(128 * 1024 * 1024)?; // 128 MB
-
+            let backend = if cfg!(feature = "maindb-backend-rocksdb") {
+                let kv = Arc::new(open_kv(
+                    path.join("db"),
+                    vec![
+                        block_storage::BlockPrimaryIndex::descriptor(&db_cache),
+                        block_storage::BlockByLevelIndex::descriptor(&db_cache),
+                        block_storage::BlockByContextHashIndex::descriptor(&db_cache),
+                        BlockMetaStorage::descriptor(&db_cache),
+                        OperationsStorage::descriptor(&db_cache),
+                        OperationsMetaStorage::descriptor(&db_cache),
+                        SystemStorage::descriptor(&db_cache),
+                        Sequences::descriptor(&db_cache),
+                        MempoolStorage::descriptor(&db_cache),
+                        ChainMetaStorage::descriptor(&db_cache),
+                        PredecessorStorage::descriptor(&db_cache),
+                        BlockAdditionalData::descriptor(&db_cache),
+                    ],
+                    &cfg,
+                )?);
+                TezedgeDatabaseBackendOptions::RocksDB(database::rockdb_backend::RocksDBBackend::from_db(kv)?)
+            } else if cfg!(feature = "maindb-backend-sled") {
+                TezedgeDatabaseBackendOptions::SledDB(database::sled_backend::SledDBBackend::new(path.join("db"))?)
+            } else {
+                TezedgeDatabaseBackendOptions::SledDB(database::sled_backend::SledDBBackend::new(path.join("db"))?)
```

Comment:
@mambisi @jurajselep 
do we want to run tests by default with `sled` or `rocksdb` ?
I would say better rocksdb, because it is default for node also,
so please change it to `rocksdb`

Reply:
Ok i will change the default to RockDB